submissions/Lu-sure/tiny-JS-world/index.js

```diff
@@ -0,0 +1,68 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Lu-sure/Lu-sure.github.io/tree/main/A_tiny_JS_world
+   Web app: https://lu-sure.github.io/A_tiny_JS_world/
+   */
+
+  const dog = {
+    species: 'dog',
+    name: 'Easy',
+    gender: 'female',
+    age: 2,
+    legs: 4,
+    hands: 0,
+    saying: 'woof-woof!',
+    friendly: 'everyone'
+  };
+ 
+  const cat = {
+    species: 'cat',
+    name: 'Roxy',
+    gender: 'female',
+    age: 1,
+    legs: 4,
+    hands: 0,
+    saying: 'meow!!',
+    friendly: 'Lila'
+  };
+ 
+  const man = {
+    species: 'human',
+    name: 'Kolia',
+    gender: 'male',
+    age: 26,
+    legs: 2,
+    hands: 2,
+    saying: 'Let`s work all day long!',
+    friendly: 'Love, Easy'
+  };
+ 
+  const woman = {
+    species: 'human',
+    name: 'Love',
+    gender: 'female',
+    age: 26,
+    legs: 2,
+    hands: 2,
+    saying: 'Absolutely agree!',
+    friendly: 'Easy, Kolia'
+  };
+ 
+  const catWoman = {
+    species: 'cat+human',
+    name: 'Lila',
+    gender: 'female',
+    age: 21,
+    legs: 2,
+    hands: 2,
+    saying: cat.saying,
+    friendly: 'Roxy, Kolia'
+  };
+ 
+ const inhabitants = [dog, cat, man, woman, catWoman];
+ 
+ const getPrintText = (obj) => Object.values(obj).join('; ');
+ 
+ inhabitants.forEach(creature => print(getPrintText(creature)));
+ 
```

Comment:
This symbol
![image](https://user-images.githubusercontent.com/8686631/100597644-5f805b80-3306-11eb-8afe-cc4c04d3cfa2.png)
normally means there is no empty line at the end of file.

Here we can see line 68 is empty yet github warns something is wrong with the end of file.
Possible reasons:
(1) line 68 contains some whitespaces and technically this is not an empty line
(2) newline chatacter does not contain LF (Unix style newline, and a part of Windows newline style CRLF), which may happen when the platform is Mac (CR newline character) and/or git newline setting is not set properly

You may want to check your environment based on [this](https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3) and set proper [newlines handling by git](https://docs.github.com/en/free-pro-team@latest/github/using-git/configuring-git-to-handle-line-endings)

**You do not need to fix this** here, but make sure to tune your IDE and git properly to avoid the issue in your future submissions.

Reply:
I found out the actual reason is (1), the line contains whitespase. Thank u))

Reply:
This is also auto-fixable with proper IDE settings. Make IDE do dirty job for you.

Reply:
Already made! Good idea)